pkg/webui/lib/components/error-view.js

```diff
@@ -39,8 +40,14 @@ class ErrorView extends React.Component {
 
   componentDidCatch(error, errorInfo) {
     Sentry.withScope(scope => {
-      scope.setExtras(errorInfo)
-      Sentry.captureException(error)
+      scope.setExtras(error)
+      if (error.message) {
+        const errorId = getBackendErrorId(error)
+        scope.setFingerprint([errorId])
+        Sentry.captureException(new Error(errorId))
+      } else {
+        Sentry.captureException(error)
+      }
```

Comment:
Better to use our error utils here to distinguish the error `isBackend` and `isFrontend`.

You might also consider making a new util `getErrorFingerprint(error)` which basically does all of the above and we can maintain it in one place.